docs/UserGuide.md

```diff
@@ -2,1012 +2,1166 @@
 
-Format:
 ```
 exit
 ```
 
-[Back to list of commands](#commands)
+[Back to list of features](#features)
+
+---
+
+## Managing Courses
+In the following sections, you will find the features that are available to you on the `Course Page`.
+A `Course Page` typically looks like this:
+
+<img src="./images/UiCoursePage.png" width="650" height="529"/>
+<br>
 
---------------------------------------------------------------------------------------------------------------
 
-## Course Page Commands
-![image](./images/UiCoursePage.png)
-###### Figure 4
 
 ### Adding a course: `add course`
+If you're teaching a new course, you can use this command to add it to the `Course Page`.
 
-Adds a course to the list of courses in the course page.
+**Command Format:**
 
-Format:
 ```
 add course <COURSE>
 ```
 
-- Adding a course with the same module code is not allowed.
-    - For example, if a course with module code `CS2103T` already exists in CLIpboard, entering `add course CS2103T` will display an error message inside the result display.
-<div markdown="block" class="alert alert-info">
-This command is case-sensitive.<br>
-e.g. <code>CS2103T</code> and <code>cs2103t</code> are treated as different courses.
+<div markdown="block" class="alert alert-info">:information_source: **Note:**<br>
+  * Adding courses with the same course code is not possible.
+  * Course codes can contain alphanumeric and special characters. e.g. "AY2223S2-CS2103T"
+  * This command is case-sensitive. e.g. "CS2103T" and "cs2103t" are treated as different courses.
 </div>
 
 Examples:
+
 - `add course CS2103T` will add a new course with module code `CS2103T`
 - `add course CS2105` will add a new course with module code `CS2105`
 
-<div markdown="span" class="alert alert-primary">:bulb: **Did you know?:**
-Course code can have special characters! i.e. "AY2223S2-CS2103T"
-</div>
+
 <br>
 
-[Back to list of commands](#commands)
+[Back to list of features](#features)
 
 ### Deleting a course: `delete course`
+If you're done teaching a course, you can remove it with this command.
 
-Deletes a course from the list of courses in the course page.
+**Command Format:**
 
-Format:
 ```
 delete course <INDEX>
 ```
 
 Examples:
-- `delete course 1` will delete the first course listed in the course list.
 
-[Back to list of commands](#commands)
+- `delete course 1` will delete the first course listed in your course list.
+
+[Back to list of features](#features)
 
 ### Editing a course: `edit course`
 
-Edits an existing course in the course list.
+Use this command to change the course code of an existing course.
+
+**Command Format:**
 
-Format:
 ```
 edit course <INDEX> <NEW COURSE NAME>
 ```
 
-- Edits the module code of the course at index `<INDEX>`.
-
 Examples:
-- `edit course 1 CS2106` will rename the first course listed in the course list to `CS2106`.
 
-[Back to list of commands](#commands)
+- `edit course 1 CS2106` will rename your first course listed in your course list to "CS2106".
```

Comment:
```suggestion
- `edit course 1 CS2106` will rename your *first* course listed in your course list to "CS2106".
```
Just a minor one for consistency. 
Have other instances as well, not adding more comments to not clutter this up as it's easily searchable.

Reply:
> Just a minor one for consistency.
> Have other instances as well, not adding more comments to not clutter this up as it's easily searchable.

just to clarify,
ive intentionally tried to be consistent to not italicisie the "first" in all examples of command formats. the only times i want to italicise them is in the sections that are before "Command Format" because the reader hasnt been introduced to what index means, so i draw extra emphasis there to show that the number for the command means something.

Reply:
Ah then that makes sense 👍, no change required.